src/apps/annotation-text/AnnotatedText/AnnotatedText.tsx

```diff
@@ -171,6 +171,7 @@ export const AnnotatedText = (props: AnnotatedTextProps) => {
 
           {props.usePopup && (
             <TextAnnotationPopup
+              asPortal
```

Comment:
Note: this will ensure identical popup move/scroll behavior with the latest versions of `@recogito/text-annotator`.